app/views/devise/shared/_social_signin.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="social-links flex space-x-2">
+  <%= button_to user_github_omniauth_authorize_path, class: "flex p-4 justify-between inline-block rounded px-6 py-2.5 text-xs font-medium uppercase leading-normal text-white shadow-md transition duration-150 ease-in-out hover:shadow-lg focus:shadow-lg focus:outline-none focus:ring-0 active:shadow-lg social-signin-link", style: "background-color: #333", data: { turbo: "false" } do %>
```

Comment:
woaa! haha Tailwind is hard sometimes... would be nice to avoid these super large list of classes with probably some custom button class?